2/contrib/jenkins/install-plugins.sh

```diff
@@ -343,14 +344,27 @@ function installedPlugins() {
     done
 }
 
+function jenkinsMajorMinorVersion() {
+    if [[ -f "$JENKINS_WAR" ]]; then
+        local version major minor
+        version="$($(alternatives --display java | awk '/family.*x86_64/ { print $1; }') -jar $JENKINS_WAR --version)"
```

Comment:
@gabemontero java is needed here is there a better way to evaluate the java-binary in the s2i assemble step?

Reply:
@bakito thanks for checking ... but we reference java in the same way in the s2i run script, though you need to account for the fact that we may be running with 32 bit

See https://github.com/openshift/jenkins/blob/master/2/contrib/s2i/run#L116-L122

Reply:
since we don't care about 32/64bit for a call like this, we should definitely just invoke java and get whatever we get.

Reply:
java is not in the path during the assemble script, that is why I copied the alternatives call.